stable/weave-scope/NOTES.txt

```diff
@@ -0,0 +1,19 @@
+After installing a Scope frontend and node agents, access the frontend 
+in your web browser.  If you used this chart to install the frontend, 
+it can be accessed in the following ways:
+
```

Comment:
Can you make the output dependent on how the user installed the chart and print the necessary commands so the user can cut and paste them?